frontend/awx/administration/instance-groups/InstanceGroupPage/hooks/useIGInstanceToolbarActions.tsx

```diff
@@ -1,13 +1,102 @@
 import { useMemo } from 'react';
-import { IPageAction } from '../../../../../../framework';
+import { IPageAction, PageActionSelection, PageActionType } from '../../../../../../framework';
 import { IAwxView } from '../../../../common/useAwxView';
 import { Instance } from '../../../../interfaces/Instance';
 import { useRunHealthCheckToolbarAction } from '../../../instances/hooks/useInstanceToolbarActions';
+import { ButtonVariant } from '@patternfly/react-core';
+import { t } from 'i18next';
+import { useAssociateInstanceToIG } from './useAssociateInstanceToIG';
+import { useParams } from 'react-router-dom';
+import { useAssociateInstanceModal } from './useAssociateInstanceModal';
+import { useDisassociateInstanceFromIG } from './useDisassociateInstanceFromIG';
+import { useGetItem } from '../../../../../common/crud/useGet';
+import { awxAPI } from '../../../../common/api/awx-utils';
+import { InstanceGroup } from '../../../../interfaces/InstanceGroup';
+import { useOptions } from '../../../../../common/crud/useOptions';
+import { OptionsResponse, ActionsResponse } from '../../../../interfaces/OptionsResponse';
 
 export function useIGInstanceToolbarActions(view: IAwxView<Instance>) {
   const healthCheckAction = useRunHealthCheckToolbarAction(view, true);
+  const associateAction = useIGInstanceAssociateToolbarAction(view);
+  const disassociateAction = useIGInstanceDisassociateToolbarAction(view);
 
   return useMemo<IPageAction<Instance>[]>(() => {
-    return [healthCheckAction];
-  }, [healthCheckAction]);
+    return [associateAction, disassociateAction, healthCheckAction];
```

Comment:
There are a couple of places in this file where we have phrases like `id ?? "`.  If we are associating or disassociating and instance to an instance group id is a required field and we should not need to use the nullish coalescing operator.  My recommendation is that if id is undefined we return out of the function asap. Let me know if I can help with this.